tests/integration/defs/perf/test_perf.py

```
@@ -920,6 +920,32 @@ def is_enc_dec(self) -> bool:
+        if self.num_loras > 0:
+            return None
+
+        # These serve requests may terminate at EOS or include image tokens,
+        # so their configured text lengths are not fixed runtime lengths.
+        if self.runtime == "serve" and (self.model_name in SPEC_DEC_MODELS or
+                                        self.model_name in SERVE_IMAGE_MODELS):
+            return None
+
+        sequence_lengths = {
+            input_len + output_len
+            for input_len, output_len in zip(self.input_lens, self.output_lens)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)